# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/MacroAssembler.h
# Commit: 5718a8d121f9
# Full Hash: 5718a8d121f95ce740a11ee443444cea990d6323
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-11-03 21:39:33
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 7: Remove Frame::tls. r=lth
#   
#   We remove Frame::tls and fix prologue/epilogue. Runtime, Ion and Baseline are ready for this move already.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D83064
# ==============================================================================

diff -r 40031ad88dbd -r 5718a8d121f9 js/src/jit/MacroAssembler.h
--- a/js/src/jit/MacroAssembler.h	Tue Nov 03 10:28:10 2020 +0000
+++ b/js/src/jit/MacroAssembler.h	Tue Nov 03 10:32:55 2020 +0000
@@ -3556,8 +3556,6 @@
   void guardStringToInt32(Register str, Register output, Register scratch,
                           LiveRegisterSet volatileRegs, Label* fail);
 
-  void loadWasmTlsRegFromFrame(Register dest = WasmTlsReg);
-
   template <typename T>
   void loadTypedOrValue(const T& src, TypedOrValueRegister dest) {
     if (dest.hasValue()) {